src/EditorFeatures/Core/InheritanceMargin/InheritanceTargetItem.cs

```diff
@@ -33,16 +33,31 @@ namespace Microsoft.CodeAnalysis.InheritanceMargin
         /// </summary>
         public readonly string DisplayName;
 
+        /// <summary>
+        /// The language of the symbol.  Used to disambiguate results when multiple targets have the same name..
+        /// </summary>
+        public readonly string LanguageName;
```

Comment:
Do we really need to differentiate between language name first, or just always do project name? For Roslyn the language name is somewhat useful since that's an implicit C#/VB hint for us, but the project name would differentiate just as well.

Reply:
would project name be a problem with multi-targeting?  E.g. you'd see two results if you multitarget .netstandard2.0 and .net6

Reply:
Personally I think I would always prefer project name even if the same language. The language name could additionally be added if the project names were the same.

Reply:
(Combine my other comment here)
How about using the image moniker instead of the string to indicate the language name?
![image](https://user-images.githubusercontent.com/24360909/165637175-8a53a7f9-adbd-4fd5-9ae1-19c244786035.png)
We can always show the language moniker, and show the project name if required?

Reply:
> Do we really need to differentiate between language name first, or just always do project name? For Roslyn the language name is somewhat useful since that's an implicit C#/VB hint for us, but the project name would differentiate just as well.

I started with project name and did *not* like it. Namely because our project names are things like `Microsoft.CodeAnalysis.CSharp.EditorFeatures.UnitTests`.  So if this is just a VB vs C# diff i would prefer that to make it simple.